sale_picking_state/models/sale_order.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (http://www.akretion.com).
+# @author Pierrick BRUN <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    picking_state = fields.Selection([
```

Comment:
Add tracking in this field
- if set manually we should add a text that tell that the state have been change manually
- if set automatically (from picking value in context) we tell that this was done automatically